README.md

```diff
@@ -273,6 +273,30 @@ GridStack.init({
 });
 ```
 
+### Resizable options
```

Comment:
this readme is about general usage (can't recall why some of the resize stuff is above). the full API doc is https://github.com/gridstack/gridstack.js/blob/master/doc/README.md and where it shiuld be (if not already)